app/components/markdown-block.js

```diff
@@ -1,43 +1,221 @@
 //https://github.com/EnCiv/enciv-home/issues/41
+//https://github.com/EnCiv/enciv-home/issues/markDownBlock-withImage#45
 import React from 'react'
 import { createUseStyles } from 'react-jss'
 import cx from 'classnames'
 import ActionButton from './action-button'
 import MarkDown from 'markdown-to-jsx'
+import * as icons from '../svgr';
 
-const TextBlock = props => {
+function Iconify(props) {
+  const { iconName, ...otherProps } = props;
+  const Icon = icons[iconName];
+  return <Icon {...otherProps} />;
+}
+
+const MarkdownBlock = props => {
+  console.log("props", props);
+   const balocks = Array.isArray(props.blocks) ? [props.blocks] : props;
+  var blockarray = [];
   const {
-    className = '', // may or may not be passed. Should be applied to the outer most tag, after local classNames
-    mode = 'light', // dark, white, see top-nav-bar for differences
+    className = '',
+    subject = '', 
+    mode = 'light',
     children = '',
-    lineWidth = 'full',
+    lineWidth = 'partial',
```

Comment:
If props need to be added, beyond what is written in the issue, it should be discussed and agreed on.  Not just for this project but for any project.  This is how other components will use this one.

The blocks prop would be implemented at a higher level, rendering multiple MarkDownBlocks. React is hierarchical, with components rendering components, rendering components. 

But I do see a challenge with the speck as written.  If we are to use MarkdownBlock as a component within a higher level component, it would be hard to render 4 across because the "...Block" components are formatted to fit the width of the screen.   It suggests that we need a MarkdownWithImage component, that fit's to the width it is given, and then MarkdownBlock could use that component, but add the padding on the sides as needed for maxPanelWidth